File: 10. Subprogram/C. Membalik yang Terbalik/c_membalik_yang_terbalik.c
#include <stdio.h>

int reverse(int x){
  int res = 0;
  while (x > 0){
    res = res * 10 + x % 10;
    x /= 10;
  }
  return res;
}

int main(){
  int A, B;
  scanf("%d %d", &A, &B);
  printf("%d", reverse(reverse(A) + reverse(B)));
  return 0;
}

File: 10. Subprogram/C. Membalik yang Terbalik/c_membalik_yang_terbalik.java
import java.util.Scanner;

class c_membalik_yang_terbalik {
    public static int reverse(int x) {
        int res = 0;
        while (x > 0) {
            res = res * 10 + x % 10;
            x /= 10;
        }
        return res;
    }

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        int A = scan.nextInt();
        int B = scan.nextInt();
        System.out.println(reverse(reverse(A) + reverse(B)));
        scan.close();
    }
}